ai/main.py

```
@@ -38,21 +43,48 @@ def synopsys(book_id: int, authorization: str = Header(None)):
 @app.post("/v1/book/{book_id}/story")
-def story(book_id: int, choice: str = Body(None), authorization: str = Header(...)):
+def story(book_id: int, request: StoryRequest, authorization: str = Header(...)):
+    try:
+        book_info = get_book(book_id, authorization)
+
+        if book_info:
+            genre = book_info.get("genre")
+            character = book_info.get("character")
+            background = book_info.get("background")
+            result = ai.generate_story(
+                genre = genre,
```

Flake8 (7.2.0)

[error] unexpected spaces around keyword / parameter equals

(E251)